modules/tracking/src/trackerKCF.cpp

```diff
@@ -87,18 +87,20 @@ namespace cv{
     * KCF functions and vars
     */
     void createHanningWindow(OutputArray _dst, const cv::Size winSize, const int type) const;
-    void inline fft2(const Mat src, std::vector<Mat> & dest) const;
-    void inline fft2(const Mat src, Mat & dest) const;
+    void inline fft2(const Mat src, std::vector<Mat> & _dest, std::vector<Mat> & _layers) const;
```

Comment:
I believe inline does not make much of a difference these days in C++. Anyway, I think it does not hurt having it either. I had never used it in methods/functions like these ones that are not precisely very short (few lines without control flow and loops).